src/validation.c

```diff
@@ -495,7 +495,7 @@ adopt_and_validate_vrf_routes(const NetplanParser *npp, GHashTable *netdefs, GEr
                 g_debug("%s: Ignoring redundant routes table %d (matches VRF table)", nd->id, r->table);
                 continue;
             } else if (r->table != NETPLAN_ROUTE_TABLE_UNSPEC) {
-                g_set_error(error, G_MARKUP_ERROR, G_MARKUP_ERROR_INVALID_CONTENT,
+                g_set_error(error, NETPLAN_VALIDATION_ERROR, NETPLAN_ERROR_CONFIG_GENERIC,
```

Comment:
Why isn't this `NETPLAN_ERROR_CONFIG_VALIDATION`, too? Or the others below `NETPLAN_ERROR_CONFIG_GENERIC`, as we don't have the filepath/line/col context here.

Reply:
_CONFIG_VALIDATION is the one that is supposed to have the information about the context. For this error we don't know the context so it should be _CONFIG_GENERIC.